credentials/java/com/google/auth/ServiceAccountSigner.java

```diff
@@ -69,7 +70,7 @@ public int hashCode() {
   /**
    * Returns the service account associated with the signer.
    */
-  String getAccount();
+  String getAccount() throws IOException;
```

Comment:
This is a breaking change, I think we should avoid this.

Reply:
Fixed